sources/Inputs/include/Configuration.h

```diff
@@ -98,6 +98,42 @@ class Configuration {
     return dsoVoltageLevel_;
   }
 
+  /**
+   * @brief Get the Time at which the simulation will start
+   *
+   * @returns the start time value
+   */
+  float getStartTime() const {
```

Comment:
We tried to use the same type that Dynawo uses to store the start/stop time in the Job definition. Should have used a `double` instead of a `float`, anyway.

Reply:
in dynawo double should have been used anyway. Nethertheless, chrono types should be used here

Reply:
Fixed, all simulation related times are managed as chrono::seconds in configuration, job definition and dynamic models parameters. Conversion to dynawo expected type is done only when preparing dynawo output files.